components/News/NewsItem.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from "react";
+import Link from "next/link";
+import { TypeNews } from "../../generated/contentful";
+import Image from "next/image";
+import Style from "./styles.module.scss";
+
+export interface NewsItemProps {
+  newsItem: TypeNews;
+}
+
+const NewsItem: FunctionComponent<
+  NewsItemProps & React.HTMLAttributes<HTMLDivElement>
+> = ({ newsItem, ...props }) => {
+  return (
+    <>
+      <div className={Style.newsLayoutItem} {...props}>
```

Comment:
This would make it omit `className={Style.newsLayoutItem}` if `className` is provided from the outside would it not?

```suggestion
      <div {...props} className={clsx(Style.newsLayoutItem, props.className)}>
```